live-examples/js-examples/array-keys.html

```diff
@@ -0,0 +1,10 @@
+<pre>
+<code id="static-js">var array1 = ['a', 'b', 'c'];
+var iterator = array1.keys(); 
+
+console.log(iterator.next()); // expected output: Object { value: 0, done: false }
+console.log(iterator.next()); // expected output: Object { value: 1, done: false }
+console.log(iterator.next()); // expected output: Object { value: 2, done: false }
+console.log(iterator.next()); // expected output: Object { value: undefined, done: true }
```

Comment:
I think it would be more natural, given an iterator, to use it in a loop, like:

```
for (let key of iterator) {
  console.log(key);
}
```

...than to call `iterator.next()` directly.

(note that then the `// expected output` lines have to change, too.)